spyder/plugins/onlinehelp/tests/test_pydocgui.py

```diff
@@ -48,6 +48,10 @@ def pydocbrowser(qtbot):
      NumpyVersion(np.__version__) < NumpyVersion('1.21.0')),
     reason="Fails on Mac and older versions of Numpy"
 )
+@pytest.mark.skipif(
+    sys.platform.startswith('linux'),
+    reason="Stalls CI frequenly on Linux"
```

Comment:
This also hangs on Windows a lot as well. So I think the only platform we should leave it is on Mac.